src/frontend/Dockerfile

```diff
@@ -42,3 +42,9 @@ RUN apt-get update && \
 # Un-privileged user running the application
 USER ${DOCKER_USER:-1000}
 
+
+# -- Production --
+FROM core as production
+
+# Un-privileged user running the application
+USER ${DOCKER_USER:-1000}
```

Comment:
Not sure this is required as this image is used for development / build system, and will not run in production.

Reply:
You're right! Removed it